src/test/java/com/hazelcast/kubernetes/KubernetesConfigTest.java

```diff
@@ -231,6 +232,41 @@ public void readFileContents()
         assertEquals(expectedContents, actualContents);
     }
 
+    @Test(expected = InvalidConfigurationException.class)
+    public void invalidCaCertificateWithKubernetesApiMode() {
+        Map<String, Comparable> properties = new HashMap<String, Comparable>();
+        properties.put(SERVICE_DNS.key(), null);
```

Comment:
You shouldn't need to specify it. I guess that Kubernetes API is the default mode.

Reply:
I added it to make it explicit in the test cases. But if preferred I will remove it.